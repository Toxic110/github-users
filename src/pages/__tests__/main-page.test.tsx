import { MainPage } from '@pages';
import { store } from '@store';
import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';

test('render main page', () => {
  const component = renderer.create(
    <Provider store={store}>
      <MainPage />
    </Provider>,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
