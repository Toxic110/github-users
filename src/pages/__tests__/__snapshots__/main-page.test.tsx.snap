// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render main page 1`] = `<users />`;
